Apps/WebClient/src/ClientApp/src/components/monthYearPicker.vue

```diff
@@ -90,25 +90,16 @@ export default class MonthYearPickerComponent extends Vue {
         return this.selectedDate.format("MMMM yyyy");
     }
 
-    private getDisplayYearCss(year: number) {
-        return `item col-12 ${
-            this.currentMonth.year() === year ? "selected" : ""
-        }`;
+    private isCurrentYear(year: number): boolean {
+        return this.currentMonth.year() === year;
     }
 
-    private getDisplayMonthCss(displayMonth: MonthToDisplay) {
-        if (displayMonth.hasData) {
-            // Months are indexed 0-11
-            const monthIndex = this.currentMonth.month() - 1;
-            return `item col-4 ${
-                monthIndex === this.monthsToDisplay.indexOf(displayMonth) &&
-                this.currentMonth.year() === this.selectedYear
-                    ? "selected"
-                    : ""
-            }`;
-        } else {
-            return "item col-4 no-data";
-        }
+    private isCurrentMonth(displayMonth: MonthToDisplay): boolean {
```

Comment:
Can you make this a getter instead of a function?

Reply:
dont think so since it needs to take the month to do the calculation